examples/staticcheck/.bazelversion

```diff
@@ -0,0 +1 @@
+6.4.0
```

Comment:
Let's not pin this here, but perhaps use the same version as the outer project with a symlink?

Reply:
I didn't find any other `.bazelversion` files in the repo, so I just dropped the current one.